src/main/scala/com/typesafe/sbt/packager/linux/LinuxPlugin.scala

```diff
@@ -249,4 +249,7 @@ object LinuxPlugin extends AutoPlugin {
     )
   }
 
+  final def chdir(installLocation: String, packageName: String): String =
+    s"$installLocation/$packageName"
+
```

Comment:
Yes, the `chdir` is actually not change directory - it's the `chdir` variable in the RPM scriptlet which is the actual directory where the application is installed.